graph/types/schema.graphqls

```diff
@@ -14,11 +14,14 @@ type Query {
 }
 
 type Mutation {
-  createDiscussion(anonymityType: AnonymityType!, title: String!): Discussion!
   addDiscussionParticipant(discussionID: String!, userID: String!): Participant!
+  addFlair(discussionID: ID!, flairID: String!): Participant!
```

Comment:
In this implementation I think it's reasonable to block on whether the current requesting user has the participantID associated with them rather than dealing with the moderator or admin auth logic in this PR.

Reply:
I think for 1, that's kinda what this line is for. If you get your available flair, then you can use this API to connect your participant in a particular discussion to one of those flairs. But maybe you're saying that you want to attach a specific flair to a participant other than yourself? I'm not sure if I'd want someone to reveal my flair for me. So this was designed to only allow a user to attach flair to their own participant. But if you want someone to assign flair for someone else, we can design for that. 

For 2, yea right now there's no way to remove available flair, even for yourself. So that needs to be added. Similar to 1, not sure if you want someone else to be able to do that for you. I would assume no, but that can also be added with some thought about who should be able to do that. 

For 3, unsetting flair is done by removing it. But similarly it can only be done for yourself....

Reply:
So there are two things that need to be accomplished:
1) add/remove flair to a user for available flair. If that's what this is doing in #1 then it should return a `User` object. Perhaps you can be more specific by calling it `addAvailableFlairToUser`. I do feel strongly that others need to manipulate this endpoint.
2) Set a participant's flair. There is a future possibility that a participant may have multiple pieces of flair, but for now it's a single piece of flair. As such, there needs to be a `Set` idea.

I think we're saying similar things here but I just want to make sure we're on the same page that there are two different ways to add or set flair -- one where it attaches to a user and one where it attaches to the participant.

Reply:
For 1, since this is creating a UserFlair object, I would think it should return the created object. Do you want to return the User object regardless?

Reply:
> If that's what this is doing in #1 then it should return a User object. 

@nedrocks there's one outstanding question here, for adding/removing flair to a user's available flair, do you want to return the flair or the user? Right now I'm returning the flair.